### DISCUSSION TITLE
Welcome to django-comments-dab Discussions!

### DISCUSSION THREAD

**Discussion by Radi85** [Radi85/Comment#116] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hossein-shahsavan** [comment]
hi how do I can change the default date and calendar into persian date???

**Reply by Radi85** [reply]
Use the correct `LANGUAGE_CODE` in your settings and follow this [guide](https://django-comment-dab.readthedocs.io/i18n.html#adding-support-for-translation) to generate translation files for your language.
